core/include/mutex.h

```diff
@@ -126,6 +126,18 @@ typedef struct {
     list_node_t queue;
 } mutex_t;
 
+/**
+ * @brief   A cancellation structure for with with @ref mutex_lock_cancelable
```

Comment:
```suggestion
 * @brief   A cancellation structure for @ref mutex_lock_cancelable
```
?

Reply:
Good catch. I think I meant "for use with".